ReactSkia/sdk/CurlNetworking.h

```diff
@@ -19,7 +19,8 @@
 #define CA_CERTIFICATE       "/etc/ssl/certs/ca-certificates.crt"      /**< The certificate of the CA to establish https connection to the server*/
 #endif
 
-#define MAX_PARALLEL_CONNECTION 5
+#define MAX_PARALLEL_CONNECTION 6L
+#define SECTOMILLSECCONVERTER(time) time*1000
```

Comment:
Use macro similar to used in Utils.h _MS suffix. Also we need one more macro for max total parallel connection